solution/backend/regulations/views/reader.py

```diff
@@ -113,6 +113,15 @@ def get_context_data(self, **kwargs):
 
         return {**context, **c, **version_info}
 
+    def get(self, request, *args, **kwargs):
+        if kwargs.get('version') is None:
+            versions = Part.objects.versions(kwargs.get("title"), kwargs.get('part'))
+            if versions is None:
+                raise Http404
+            kwargs['version'] = versions[0]['date']
+            return HttpResponseRedirect(reverse('reader_view', kwargs=kwargs))
+        return super().get(request, *args, **kwargs)
```

Comment:
fix issue that showed Version Picker by default when linking to a Subpart without explicitly passing in a date in the URL.  This now chooses the most recent version date and uses it, which no longer shows the Version Picker by default.